docs/FAQ.md

```diff
@@ -20,9 +20,13 @@ process, we try to stay on top of these changes and release updates as quickly a
 The RAPIDS Accelerator for Apache Spark officially supports:
 - [Apache Spark](get-started/getting-started-on-prem.md)
 - [AWS EMR 6.2+](get-started/getting-started-aws-emr.md)
-- [Databricks Runtime 7.3, 9.1](get-started/getting-started-databricks.md)
+- [Databricks Runtime 9.1, 10.4](get-started/getting-started-databricks.md)
 - [Google Cloud Dataproc 2.0](get-started/getting-started-gcp.md)
 - [Azure Synapse](get-started/getting-started-azure-synapse-analytics.md)
+- Cloudera provides Spark RAPIDS plugin packaged through 
```

Comment:
Recommend we rewrite the cloudera one to be:
```
- [CDS 3.2](https://docs.cloudera.com/cdp-private-cloud-base/7.1.7/cds-3/topics/spark-spark-3-overview.html)
```
The requirements link is the second bullet point in the CDS documentation after overview, Ia m not sure we need to call it out.

Reply:
I am fine with that. It is just different than the other bullet points, but that's minor.